src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/VarHandleNames.kt

```diff
@@ -20,7 +20,7 @@ import java.lang.reflect.Field
 /**
  * Helper object to provide the field name and the owner of the VarHandle method call.
  */
-@Suppress("SameParameterValue")
+@Suppress("SameParameterValue", "Since15")
```

Comment:
Why is `Since15` necessary here?

Reply:
To suppress Idea warnings about usage of `since JDK 9` API.

Reply:
How can `since JDK 9 API` usages be in the code with the source level being Java 8?